apps/ledger-live-mobile/src/components/RootDrawer/RootDrawer.tsx

```diff
@@ -9,6 +9,7 @@ import { useNavigation } from "@react-navigation/native";
 import { StackNavigationProp } from "@react-navigation/stack";
 import { NavigatorName } from "../../const";
 import { PTXServicesAppleWarning } from "./InitialDrawers/PTXServicesAppleWarning";
+import { EarnInfoDrawer } from "../../screens/PTX/Earn/EarnInfoDrawer";
```

Comment:
Should this (and the id const `EarnInfoDrawerID`) be used in the RootDrawerSelector fn (line 33)? Or if not, can we remove the import?